src/main/java/net/gensokyoreimagined/motoori/KosuzuParsesEverything.java

```diff
@@ -43,18 +44,20 @@ public KosuzuParsesEverything(Kosuzu kosuzu) {
         }
 
         kosuzu.getLogger().info("Prepared " + this.regexes.size() + " regexes");
+
+        syntaxBlacklist = config.getStringList("match.blacklist");
+
+        kosuzu.getLogger().info("Added " + syntaxBlacklist.size() + " blacklist entries");
     }
 
     /**
-     * Extracts the text message from a chat component
+     * Extracts the actual text message from a chat message
      * Also determines if we should translate the message
      * @param component The chat component created from the message
      * @param player The player who sent the message
      * @return The text message, or null if it could/should not be translated
      */
-    public @Nullable String getTextMessage(Component component, Player player) {
-        var text =  PlainTextComponentSerializer.plainText().serialize(component);
-
+    public @Nullable String getTextMessage(String text, Player player) {
```

Comment:
I guess this goes with my previous comment - we could simply do the unwanted matching here and not even need to change the signature of the method.

Reply:
I wanted to keep it separate to avoid possible issues with assumptions about what functions do or don't do. If the job of this function is to get the actual original message, then it shouldn't modify the message.

However, I can return it to using the `Component` if preferred. That change was made since it made the code that called the function more streamlined. Since it was the ONLY place that called it, I thought it'd be okay to change the signature.

Reply:
You passed a `Component` in the `actualMessage = parser.removeUnwantedSyntax(message);` which was why I originally asked, which now I've tested, doesn't compile as well.